mrtt-ui/src/components/FakeSite.js

```diff
@@ -1,23 +1,15 @@
-import { Link, Stack, Typography } from '@mui/material'
-import { Link as LinkReactRouter } from 'react-router-dom'
+import { Link } from '../styles/typography'
+import { Stack, Typography } from '@mui/material'
 import React from 'react'
 
 const FakeSite = () => {
   return (
     <Stack>
       <Typography variant='h4'>Fake Site for Demo</Typography>
-      <Link component={LinkReactRouter} to='/1/form/project-details'>
-        Project Details
-      </Link>
-      <Link component={LinkReactRouter} to='/1/form/site-background'>
-        Site Background
-      </Link>
-      <Link component={LinkReactRouter} to='/1/form/restoration-aims'>
-        Restoration Aims
-      </Link>
-      <Link component={LinkReactRouter} to='/1/form/causes-of-decline'>
-        Causes of Decline
-      </Link>
+      <Link to='/1/form/project-details'>Project Details</Link>
+      <Link to='/1/form/site-background'>Site Background</Link>
+      <Link to='/1/form/restoration-aims'>Restoration Aims</Link>
+      <Link to='/1/form/causes-of-decline'>Causes of Decline</Link>
```

Comment:
I thought I was going to use links for this ticket, so simplified these thinking I was going to usel the new Link component. Will use elsewhere likely, and even if not, I like this better.